openpype/plugins/publish/extract_review_slate.py

```diff
@@ -215,7 +283,8 @@ def process(self, instance):
                     width_scale, height_scale, to_width, to_height,
                     width_half_pad, height_half_pad
                 )
-
+            # add output frame rate as a filter, just in case
+            scaling_arg += ",fps={}".format(input_frame_rate)
```

Comment:
Variable `scaling_arg` does not exist all the time. I don't uderstand why it didn't crash anywehere yet.

Reply:
Right. Indent error. It didn't crash yet 'cause reformat is almost always in the tags. Will fix that